### PR TITLE
v1.1.0

### PR DESCRIPTION
**Pull request by connor-baer**

### CODE REVIEW

File: src/util/shared-prop-types.js [hunk 1]
```diff
@@ -14,12 +14,33 @@
...
+) => {
+  if (props[propName] !== null) {
+    deprecate(
+      // eslint-disable-next-line max-len
+      `"${propName}" prop of "${componentName}" has been deprecated.\n${explanation}`
+    );
+  }
+
+  return PropTypes.checkPropTypes(
+    { propName: propType },
```
Comment (voronianski) [inline src/util/shared-prop-types.js]: @connor-baer just noticed this, shouldn't it be `{ [propName]: propType }`?

Comment (connor-baer) [reply src/util/shared-prop-types.js]: You're right! I copied it from another custom prop type where it's also wrong. I'll make a PR to fix it 👍

Comment (connor-baer) [reply src/util/shared-prop-types.js]: Here you go: https://github.com/sumup/circuit-ui/pull/449

Comment (voronianski) [reply src/util/shared-prop-types.js]: approved1

**Pull request merged**